gspread/worksheet.py

```diff
@@ -2607,6 +2607,32 @@ def unmerge_cells(self, name):
 
         return self.spreadsheet.batch_update(body)
 
+    def get_notes(self, default_empty_value: str = None) -> List[List[str]]:
```

Comment:
This typing won't pass, either you should set the defautl value to `""` an empty string, change the typing from `str` to `Optional[str]`.

Reply:
Yeah. Sorry I ignored it. Now I have changed the typing to Optional[str]